### PR TITLE
Fix #210

### LINKED ISSUE #210: There is no way now to make a comment in RakuScript
Just like in the tile I think it for now we can add one line comments as lines that starts from `#`

### PR DESCRIPTION
**Pull request by Jeremi360**
This fixes / implements #210

### CODE REVIEW

File: addons/Rakugo/lib/systems/Parser.gd [hunk 1]
```diff
@@ -67,12 +67,12 @@ var parser_regex :={
 	# jump label
 	JUMP = "^jump (?<label>{NAME})( if (?<expression>.+))?$",
 	# for setting Rakugo variables
-	SET_VARIABLE = "^(?<lvar_name>{VARIABLE}) = ((?<text>{STRING})|(?<number>{NUMERIC})|(?<rvar_name>{VARIABLE}))$",
+	SSET_VARIABLE = "^(?<lvar_name>{VARIABLE}) = ((?<text>{STRING})|(?<number>{NUMERIC})|(?<rvar_name>{VARIABLE}))$",
+	COMMENT = "^#.*$",
```
Comment (theludovyc) [inline addons/Rakugo/lib/systems/Parser.gd]: There is a way to not use regex and to do not read a line start by '#' (you can use a func from class string to remove space or tab if it needed), plus it is more efficient :) !

A clue (do not read just this line, start from 151) : https://github.com/rakugoteam/Rakugo-Dialogue-System/blob/bcb65639c820bc414411bee3e07a4d35523e8def/addons/Rakugo/lib/systems/Parser.gd#L154

**Review by theludovyc — CHANGES_REQUESTED** [review]

File: addons/Rakugo/lib/systems/Parser.gd (hunk 1 above)
Comment (Jeremi360) [reply addons/Rakugo/lib/systems/Parser.gd]: Okey I will update code in this way.

Comment (Jeremi360) [reply addons/Rakugo/lib/systems/Parser.gd]: Just fixed with my last commit

Comment (Jeremi360) [reply addons/Rakugo/lib/systems/Parser.gd]: Just fixed with my last commit @theludovyc

**Comment by theludovyc** [comment]
Fix #210

File: addons/Rakugo/lib/systems/Parser.gd (hunk 1 above)
Comment (theludovyc) [reply addons/Rakugo/lib/systems/Parser.gd]: Yeah you find it ! 🥳

**Review by theludovyc — APPROVED** [review]

**Comment by theludovyc** [comment]
Ready for merge :) !

**Pull request merged**